lib/Image/Adapter/Imagick.php

```diff
@@ -279,7 +286,7 @@ public function save(string $path, string $format = null, int $quality = null):
         }
 
         if ($realTargetPath) {
-            File::rename($path, $realTargetPath);
+            $this->filesystem->rename($path, $realTargetPath);
```

Comment:
Should we pass the "override" flag here? per symfony default, it won't override an existing target!